README.md

```diff
@@ -141,6 +141,12 @@ sudo systemctl restart pve-cluster.service pvedaemon.service pvestatd.service pv
 
 - Verify Network Connectivity: Ensure that the Proxmox VE nodes can reach the Pure Storage array over the network. Check for firewall rules or network issues that might be blocking communication.
 - Review Logs: Check the Proxmox VE logs for any error messages related to storage or the plugin. Logs are typically found in /var/log/pve.
+  These commands are helpful for troubleshooting:
+  ```bash
+  journalctl -u multipath #diagnose issues with the multipath service
+  ls /etc/iscsi/nodes/ #list what iscsi nodes are mounted
+  cat /etc/multipath/wwids #list wwids of discs on the system
+  ```
```

Comment:
`cat /etc/multipath/wwids` shows only catalog
correct way to show multipath disks is `multipath -ll`, also lsblk shows disks
`ls -l /dev/disk/by-id/` shows symlink by wwn
`ls -l /dev/mapper/3624a9370*` shows by wwn